dll/win32/shell32/shfldr.h

```diff
@@ -27,11 +27,24 @@
 #define CHARS_IN_GUID 39
 
 typedef struct {
-    int colnameid;
-    int pcsFlags;
-    int fmt;
-    int cxChar;
+    WORD colnameid; // Column title text resource id passed to LoadString
+    WORD colstate;  // SHCOLSTATEF returned by IShellFolder2::GetDefaultColumnState (HACK: SHCOLSTATEF truncated to WORD to reduce .rdata section size)
```

Comment:
```suggestion
    WORD colstate;  // SHCOLSTATEF returned by IShellFolder2::GetDefaultColumnState
                    // HACK: SHCOLSTATEF truncated to WORD to reduce .rdata section size
```
(or however you want to format this line to be under 100 characters)